data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/LivenessTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beaconrestapi.handlers.tekuv1.admin;
+
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import io.javalin.http.Context;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class LivenessTest {
+
+  private Liveness handler;
+
+  @BeforeEach
+  public void setup() {
+    handler = new Liveness();
+  }
+
+  private Context context = mock(Context.class);
+
+  @Test
+  public void shouldReturnOkWhenTekuIsUp() throws Exception {
+    handler.handle(context);
+    verify(context).status(SC_OK);
```

Comment:
once the header is in, we should verify it got set here, like the SC_OK.

Reply:
done.